distribution/systemd/vector@.service

```diff
@@ -0,0 +1,23 @@
+[Unit]
+Description=Vector - %i
+Documentation=https://vector.dev
+After=network-online.target local-fs.target remote-fs.target time-sync.target
+Requires=network-online.target local-fs.target remote-fs.target time-sync.target
```

Comment:
The existing unit file doesn't include all of these, should we back port these changes?

Reply:
I think it could be a good idea. I will add those modifications.